### PR TITLE
Adds existing test files from webapp

### PR DESCRIPTION
**Pull request by steam**
All existing test files in web app related to the current brains files have been added to this project. We're going to wait until the editorials epic is merged into master on web app before removing the files over there to prevent another potentially tedious web app rebase party.

### CODE REVIEW

**Review by rynbyjn — APPROVED** [review]
LGTM, but I wonder if we should figure out a good way of exposing parts of the test folder (like stubs.js) so it can be used across both projects as well. Maybe pull it into `/src` somewhere? I don't think much else would get used from the `/test` folder across projects..

**Comment by steam** [comment]
Interesting idea about sharing helpers between the projects. Should we figure that out before we merge? Seems fine to me if it is in a future PR.

**Comment by rynbyjn** [comment]
Yeah totally in the future, just wanted to start thinking about it, but it definitely makes sense that we would want the stubs to stay up to date across projects.

**Pull request merged**
